src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -41,45 +37,54 @@ public void RelativeSearchPath_Is_Null()
             Assert.Null(AppDomain.CurrentDomain.RelativeSearchPath);
         } 
 
-
         [Fact]
         public void UnhandledException_Add_Remove()
         {
-            AppDomain.CurrentDomain.UnhandledException += new UnhandledExceptionEventHandler(MyHandler);
-            AppDomain.CurrentDomain.UnhandledException -= new UnhandledExceptionEventHandler(MyHandler);
+            RemoteInvoke(() => {
+                AppDomain.CurrentDomain.UnhandledException += new UnhandledExceptionEventHandler(MyHandler);
+                AppDomain.CurrentDomain.UnhandledException -= new UnhandledExceptionEventHandler(MyHandler);
+                return SuccessExitCode;
+            }).Dispose();
         }
 
         [Fact]
         public void UnhandledException_NotCalled_When_Handled()
         {
-            AppDomain.CurrentDomain.UnhandledException += new UnhandledExceptionEventHandler(NotExpectedToBeCalledHandler);
-            try {
-                throw new Exception();
-            }
-            catch
-            {
-            }
-            AppDomain.CurrentDomain.UnhandledException -= new UnhandledExceptionEventHandler(NotExpectedToBeCalledHandler);
+            RemoteInvoke(() => {
+                AppDomain.CurrentDomain.UnhandledException += new UnhandledExceptionEventHandler(NotExpectedToBeCalledHandler);
+                try
+                {
+                    throw new Exception();
+                }
+                catch
+                {
+                }
+                AppDomain.CurrentDomain.UnhandledException -= new UnhandledExceptionEventHandler(NotExpectedToBeCalledHandler);
+                return SuccessExitCode;
+            }).Dispose();
         }
 
         [ActiveIssue(12716)]
         [PlatformSpecific(~TestPlatforms.OSX)] // Unhandled exception on a separate process causes xunit to crash on osx
         [Fact]
         public void UnhandledException_Called()
         {
-            System.IO.File.Delete("success.txt");
-            RemoteInvokeOptions options = new RemoteInvokeOptions();
-            options.CheckExitCode = false;
-            RemoteInvoke(() =>
-            {
-                AppDomain.CurrentDomain.UnhandledException += new UnhandledExceptionEventHandler(MyHandler);
-                throw new Exception("****This Unhandled Exception is Expected****");
+            RemoteInvoke(() => {
+                System.IO.File.Delete("success.txt");
```

Comment:
I did run uap tests before any additional changes and they were passing and did not run them after netfx changes since that requires clean build.

It is probably due to `[ActiveIssue(12716)]` I haven't spotted this. I have updated all tests which I noticed that modify any static state so that they use RemoteInvoke.

I'd leave this for the person fixing this issue since there might be more uap fixes we need here. I'll leave the comment in that issue